### PR TITLE
Add a join command

### PR DESCRIPTION
**Pull request by jasin**
This works in-game. Hopefully you can use this, very simple nothing extra

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -12,8 +12,10 @@ prefix: "[BedWars]"
 permissions:
   bedwars.command:
     description: Base perm for bedwars commands
-    default: op
+    default: true
```
Comment (inxomnyaa) [inline plugin.yml:15]: nope

File: src/xenialdan/BedWars/commands/BedwarsCommand.php [hunk 2]
```diff
@@ -49,7 +49,27 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
...
+			if (is_null($arena = Loader::getInstance()->getArenas()[$args[1]]??null)) {
+			    $sender->sendMessage(TextFormat::RED . "Arena " . $args[1] . " not found");
+			    return true;
+			}
+			if (!$arena->joinTeam($sender)) {
+			    $sender->sendMessage(TextFormat::RED . "Error joining arena");
+			    return true;
+			}
+			break;
+		    }
```
Comment (inxomnyaa) [inline src/xenialdan/BedWars/commands/BedwarsCommand.php]: Mixed tabs and spaces in front

**Review by inxomnyaa — CHANGES_REQUESTED** [review]
I will battletest this, and maybe i will change some stuff  -but looks ok so far

File: plugin.yml (hunk 1 above)
Comment (jasin) [reply plugin.yml:15]: Please explain why? When OP no one can run the other commands without having permission for bedwars.command. This needlessly requires you to negate permissions for everything you don't want a user to run even though some of those commands aren't OP required

unless I am missing something. Which is a possibility

Comment (inxomnyaa) [reply plugin.yml:15]: Just realized - someone reported that issue once, it is correct to use "true". #blameshoghi for the bad permission design in PMMP

**Comment by jasin** [comment]
I think I got the tab/space issue fixed. I amended last commit and force-pushed. Let me know

**Review by inxomnyaa — APPROVED** [review]

**Pull request merged**